sabelotodo/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column, Integer, String, Boolean, ForeignKey, UniqueConstraint
+from sqlalchemy.orm import relationship
+from sabelotodo.database import Base
+
+class Item(Base):
+
+    # A to-do item. Items can form trees, with each storing the id of its
+    # parent if it has one. The order field represents the order among
+    # siblings, and in any set of siblings, order numbers must be unique.
+
+    __tablename__ = 'items'
+    __table_args__ = (UniqueConstraint('order','parent_id',name='sibling_order'),)
+    id = Column(Integer, primary_key=True)
+    name = Column(String(256))
+    order = Column(Integer, unique=True)
```

Comment:
Looks like order is unique on it's own still, we'll want to remove that, I think.

Reply:
Oh heck, good catch.